backend/src/api.py

```diff
@@ -228,9 +232,14 @@ def load_nodes(self, current_file: str):
                     except ImportError as e:
                         import_errors.append(e)
                     except RuntimeError as e:
-                        logger.warning(f"Failed to load {module}: {e}")
+                        logger.warning(f"Failed to load {module} ({file_path}): {e}")
                     except ValueError as e:
-                        logger.warning(f"Failed to load {module}: {e}")
+                        logger.warning(f"Failed to load {module} ({file_path}): {e}")
+                    except TypeMismatchError as e:
+                        type_errors.append(e)
+
+        if len(type_errors) > 0:
+            raise RuntimeError(type_errors)
```

Comment:
Right now, we just get this in the logs:
```
RuntimeError: [TypeMismatchError("Error in chainner:utility:text_append: Input type of Text A '<class 'str'>' is not assignable to varargs type 'typing.Optional[str]'"), TypeMismatchError("Error in chainner:image:z_stack: Input type of Image A '<class 'numpy.ndarray'>' is not assignable to varargs type 'typing.Optional[numpy.ndarray]'")]
```

This isn't readable at all, so I think that we should always log type errors.

```suggestion
                    except TypeMismatchError as e:
                        logger.error(e)
                        type_errors.append(e)

        if len(type_errors) > 0:
            raise RuntimeError(f"Type errors occurred in {len(type_errors)} node(s)")
```